cmd/format-xl.go

```diff
@@ -698,6 +698,22 @@ func initStorageDisks(endpoints EndpointList) ([]StorageAPI, error) {
 	return storageDisks, nil
 }
 
+// Runs through the faulty disks and record their errors.
+func initDisksWithErrors(endpoints EndpointList) ([]StorageAPI, []error) {
+	storageDisks := make([]StorageAPI, len(endpoints))
+	var dErrs = make([]error, len(storageDisks))
+	for index, endpoint := range endpoints {
+		storage, err := newStorageAPI(endpoint)
+		if err != nil && err != errDiskNotFound {
+			logger.LogIf(context.Background(), err)
+			dErrs[index] = errFaultyDisk
```

Comment:
I don't think replacing all errors with errFaultyDisk is a good idea. I think that a better idea is to map errDiskNotFound to offline node and map other "unknown" errors with corrupted

Reply:
@vadmeste is errFaultyDisk not making sense here? I mean, is it OK to consider all the other unknown errors as `errCorruptedFormat`?

Reply:
> is it OK to consider all the other unknown errors as `errCorruptedFormat`?

I think so.. because we already have only three state, online - offline - something wrong (corrupted)